### PR TITLE
Feat: Add collection information

### LINKED ISSUE #108: [Task]: Add collection information to NFTs section
**Task description.**
If collection's father NFT has infornation about it, we should display it

image

**Requirements.**
- Collection section
- Collection Name
- Collection Image

### PR DESCRIPTION
**Pull request by cpl121**
Fixes #108

## Changes

  - Add collection section, name and image
  
### Checks

* [X] Have you catchup your branch with the latest state in base?
* [X] Have you lint your code locally prior to submission?
* [X] Have you reviewed your proposed changes and removed debris?
* [X] Have you successfully ran due tests with your changes?

CM id to test success case in devnet: `4CQBaAFnAsaydZLNiQPeomUASbWLKgcS1UdjzHNQgGRd`

### CODE REVIEW

File: components/NftCard.tsx [hunk 1]
```diff
@@ -17,13 +18,15 @@ const NftCard: FC<{ title: string; imageLink: string; hash?: string }> = ({ titl
                 className='border rounded-3 mb-2 overflow-hidden d-flex flex-items-center flex-justify-center'
                 style={{ height: '168px', width: '168px' }}
             >
-                <img alt={title} src={imageLink} />
+                <div style={{ width: '100%', height: '100%', position: 'relative' }}>
```
Comment (evavirseda) [inline components/NftCard.tsx:21]:
```suggestion
                <div className='width-full height-full position-relative'>
```

File: components/NftDetails.tsx [hunk 2]
```diff
@@ -14,6 +14,11 @@ const NftDetails: FC<{ nft: Nft }> = ({ nft }) => {
                     <strong>Description</strong>: {nft.description}
                 </span>
             )}
+            {nft.mint && (
```
Comment (evavirseda) [inline components/NftDetails.tsx:17]:
```suggestion
            {nft?.mint && (
```

**Review by evavirseda — CHANGES_REQUESTED** [review]
Looks great, just two changes!

**Review by evavirseda — APPROVED** [review]
Nice job :rocket:

**Review by VmMad — APPROVED** [review]
Looks good to me, just conflicts

**Pull request merged**